modules/overview-of-distributed-workloads.adoc

```diff
@@ -0,0 +1,32 @@
+:_module-type: CONCEPT
+
+[id='overview-of-distributed-workloads_{context}']
+= Overview of distributed workloads
+
+[role='_abstract']
+You can use the distributed workloads feature to queue, scale, and manage the resources required to run data science workloads across multiple nodes in an OpenShift cluster simultaneously. Data science workloads include several types of artificial intelligence (AI) workloads, including machine learning (ML) and Python workloads.
```

Comment:
The last statement seems a bit absolute. Is that always true? Nonetheless, I suggest starting that statement with "Typically,..."